app/services/search/tag.rb

```diff
@@ -16,6 +16,24 @@ def find_document(tag_id)
         SearchClient.get(id: tag_id, index: INDEX_ALIAS)
       end
 
+      def search(query_string)
+        SearchClient.search(
+          index: INDEX_ALIAS,
+          body: {
+            query: {
+              query_string: {
+                query: query_string,
+                analyze_wildcard: true,
+                allow_leading_wildcard: false
```

Comment:
Is this because leading wildcards are a lot less performant? Or maybe there isn't a use case where we _really_ need this ability?

Reply:
Leading wildcards are WAY less performant and any ES person will always tell you to avoid them at all costs. Honestly, on small indexes they are not that bad, but when you get a big index 😬. One of the big lessons I learned from ES training is "Dont let your users take you down" and that means putting things like this in so if someone enters the tag "*tag" we stop that search from happening.